test/contrib/automotive/ecu_am.uts

```diff
@@ -18,36 +18,35 @@ iface1 = "vcan1"
 ISOTP_KERNEL_MODULE_AVAILABLE = False
 def exit_if_no_isotp_module():
     if not ISOTP_KERNEL_MODULE_AVAILABLE:
-        err = "TEST SKIPPED: can-isotp not available"
```

Comment:
Reminder: This changes have to be reverted. The purpose of subprocess.call was to display this message in the output of UTScapy. sys.stderr is redirected, and will not fulfill this purpose.